browser/src/layer/tile/CanvasTileLayer.js

```diff
@@ -2100,7 +2100,7 @@ L.CanvasTileLayer = L.Layer.extend({
 
 		var sameAddress = oldCursorAddress.equals(app.calc.cellAddress.toArray());
 
-		var isFollowingOwnCursor = parseInt(app.getFollowedViewId()) === parseInt(this._viewId);
+		var isFollowingOwnCursor = parseInt(app.getFollowedViewId()) === parseInt(this._viewId) || this._map._docLayer._searchRequested;
```

Comment:
`_searchRequested` was now removed

Reply:
we have better approach: we just always jump to result when search result arrives - that allows to remove hack with that state variable

Reply:
done